provisioning/Build.ps1

```diff
@@ -30,7 +30,7 @@ Import-Module $bootstrapModule -Verbose -Force
 # # Run all the windows provisioning scripts
 # ############
 # TODO: Create function for downloading scripts and executing them.
-$windowsBootstrapURL = "https://raw.githubusercontent.com/GPII/windows/master/provisioning"
+$windowsBootstrapURL = "https://raw.githubusercontent.com/JavierJF/windows/GPII-4231/provisioning"
```

Comment:
Please can you fix this back to point to GPII master which is most likely the cause of the CI failures

Reply:
Done, but it fails way before that, on the line that launches grunt lint, take a look, very strange error:
[](https://ci.gpii.net/job/gpii-app-code-analysis/106/console)

Reply:
And also the Color Vision itself its not working with master's provisioning script. It changes DPIScale, but not the Color Vision filter!